web/src/main/java/org/fao/geonet/monitor/onlineresource/OnlineResourceCheckerWps.java

```diff
@@ -113,7 +102,7 @@ private String getRequestXml(String time) {
                 "    <wps:Input>\n" +
                 "      <ows:Identifier>subset</ows:Identifier>\n" +
                 "      <wps:Data>\n" +
-                "        <wps:LiteralData>TIME," + literalTime + "," + literalTime + ";LATITUDE,-31.6855,-31.6855;LONGITUDE,114.8291,114.8291</wps:LiteralData>\n" +
+                "        <wps:LiteralData>TIME,1415-25-10T00:00:00.000Z,1415-25-10T00:00:00.000Z;LATITUDE,-31.6855,-31.6855;LONGITUDE,114.8291,114.8291</wps:LiteralData>\n" +
```

Comment:
Looks to me like GoGoDuck will throw an exception if no files are selected for aggregation (https://github.com/aodn/geoserver-build/blob/master/src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java#L180), so not sure how this will work.

Reply:
[request](https://github.com/aodn/core-geonetwork/files/1425715/wps-gogoduck-synchronous.txt) -> [response](https://github.com/aodn/core-geonetwork/files/1425720/response.txt)